dfttopif/parsers/vasp.py

```diff
@@ -261,14 +263,33 @@ def get_forces(self):
             conditions=Value(name="positions", vectors=self.atoms.positions.tolist())
         )
 
+    @staticmethod
+    def _get_bandgap_from_bands(energies, nelec):
```

Comment:
Here is where the real logic lives.  Its not smart enough to distinguish between direct and indirect.  CC: @paulsaxe